test/browser/logging-methods.js

```diff
@@ -146,6 +146,25 @@
         sandbox.restore()
       })
 
+      it('adds padding (if specified) to short logger names', function () {
+        try {
+          var abc = new Logdown({prefix: 'abc'})
+          var demo = new Logdown({prefix: 'demo', alignOuput: true})
+          var longDemo = new Logdown({prefix: 'longDemo', alignOuput: true})
+          var longerDemo = new Logdown({prefix: 'longerDemo', alignOuput: true})
+
+          assert(abc.prefix.length === 3)
+          assert(demo.prefix.length === 10)
+          assert(longDemo.prefix.length === 10)
+          assert(longerDemo.prefix.length === 10)
+        } catch (error) {
+          sandbox.restore()
+          throw error
+        }
+
+        sandbox.restore()
+      })
+
```

Comment:
We should on the server implementation as well
One key feature of logdown.js is that it runs on both environments seamlessly